<launch>

	<node name="object_detection_lidar" pkg="preprocessing" output="screen" type="obj_detector_node">
		<param name="input_cloud" value="/velodyne_points"/>
		<param name="output_cloud" value="/output_cloud"/>
		<param name="obj_detected_list" value="/usv_perception/lidar/objects_detected"/>
		<param name="view_pointcloud" type="bool" value="true"/>
	</node>

</launch>